src/main/java/github/scarsz/discordsrv/util/DiscordUtil.java

```diff
@@ -505,11 +505,11 @@ public static String convertRoleToMinecraftColor(Role role) {
 
     /**
      * Get a formatted String representing all of the Member's roles, delimited by DiscordToMinecraftAllRolesSeparator
-     * @param member The Member to retrieve the roles of
+     * @param List<Role> The list of roles a member has and filtered according to the whitelist or blacklist
      * @return The formatted String representing all of the Member's roles
```

Comment:
"all of the Member's roles" (also on line 507)

Reply:
I missed that, thank you